File: codevita2016/mockvita/2.java
import java.util.*;

public class Student {
    
    public static void main (String[] args){
        Scanner d= new Scanner(System.in);
        try{
                      
                double amount=0.0;       
                double p = Double.parseDouble(d.next().trim());
                p= (double)Math.round(p * 100000000000d) / 100000000000d;
                double r = Double.parseDouble(d.next().trim());
                r= (double)Math.round(r * 100000000000d) / 100000000000d;
                int t = Integer.parseInt(d.next().trim());
                if(t<=0 || p<=0|| r<=0)
                    throw new Exception();
                for(int i=0;i<t;i++){
                    amount += p + ((amount+p)*r)/1200.0;
                }
                long ans= Math.round(amount);
                System.out.println(ans);
                
             
        }catch(Exception e){
            System.out.println("Invalid Input");
        }
    }
}

File: codevita2016/mockvita/2.c
#include<math.h>
#include <stdlib.h>
#include<ctype.h>
int main(){
    int t, i;
    char str[20];
    long double p, r;
    scanf("%llf",&p);
    scanf("%llf",&r);
    scanf("%s",&str);
    for(i=0;str[i]!='\0';i++){
        if(!isdigit(str[i])){
            printf("Invalid Input\n");
            return 0;
    }
    }

    t=atoi(str);
    if(t<=0 || p<=0 || r<0){
            printf("Invalid Input\n");
            return 0;
    }

    long double amount=0;
    for(i=0;i<t;i++){
        amount += p + ((amount+p)*r)/1200.0;
    }
    i= round(amount);
    printf("Final_Amount %Lf\n",(double)i);
    return 0;
}
